pkg/launcher/chrome_profile.go

```diff
@@ -12,24 +17,140 @@ type ChromeProfile struct {
 	// which we override to put Granted profiles in a specific folder
 	// for easy management.
 	UserDataPath string
+
+	BrowserType string
 }
 
 func (l ChromeProfile) LaunchCommand(url string, profile string) []string {
-	profileName := chromeProfileName(profile)
+	profileName, _ := FindBrowserProfile(profile, l.BrowserType)
+	// profileName = chromeProfileName(profileName)
+
 	return []string{
 		l.ExecutablePath,
-		"--user-data-dir=" + l.UserDataPath,
+		// "--user-data-dir=" + l.UserDataPath,
 		"--profile-directory=" + profileName,
 		"--no-first-run",
 		"--no-default-browser-check",
 		url,
 	}
 }
 
-func chromeProfileName(profile string) string {
-	h := fnv.New32a()
-	h.Write([]byte(profile))
+// func chromeProfileName(profile string) string {
+
+// 	h := fnv.New32a()
+// 	h.Write([]byte(profile))
+
+// 	hash := fmt.Sprint(h.Sum32())
+// 	return hash
+// }
+
+// todo: verify these are correct
+var BravePathMac = "Library/Application Support/BraveSoftware/Brave-Browser/Local State"
```

Comment:
Looks like a bit of cleaning is required around here 😅

Reply:
@JoshuaWilkes when you verified all the locations on each OS did you do this manually or through Googling each location. Can only verify OSX until next year when I have access to a Windows + linux PC

Reply:
@meyerjrr I was able to verify all of them manually, as I have access to all 3